### PR TITLE
back link changes for warning screen

### PR DESCRIPTION
**Pull request by mattch1**
Back links to all screens except the two authentication screens:
login
authentication code

A typical back link journey would be:
still-required -> warning -> check-company -> company-number -> company-required (start screen)

Changes here are to add the warning screen to the back link journey

### CODE REVIEW

File: src/controllers/still.required.controller.ts [hunk 1]
```diff
@@ -22,9 +22,12 @@ const validators = [
  */
 export const getRoute = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
-  const companyName: string = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyName;
+  const company: PTFCompanyProfile = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE);
+  const companyName: string = company.companyName;
+  const backLinkUrl: string = `/company-required/company/${company.companyNumber}/warning`;
```
Comment (markpit) [inline src/controllers/still.required.controller.ts:27]: can this go i the Templates file? to avoid hard coding urls

Comment (markpit) [reply src/controllers/still.required.controller.ts:27]: or just keep the url in the html and just pass the companyNumber into the html

Comment (markpit) [reply src/controllers/still.required.controller.ts:27]: further down in this same file on line 56 it uses the pageUrls, you should do the same

const url = COMPANY_REQUIRED_CONFIRMATION.replace(":companyNumber", companyProfile.companyNumber);

Comment (mattch1) [reply src/controllers/still.required.controller.ts]: Removed hardcoding from the code - added this to the url file and as suggested replace :companyNumber.

**Pull request merged**